tooling/reproducible/repro_common.sh

```diff
@@ -332,7 +332,7 @@ function removeGeneratedClasses() {
   local JDK_DIR="$1"
   local OS="$2"
 
-  if [[ "$OS" =~ CYGWIN* ]]; then
+  if [[ "$OS" =~ CYGWIN* ]] || [[ "$OS" =~ Darwin* ]]; then
     rm -rf "$JDK_DIR/bin/server/classes.jsa"
```

Comment:
I suspect the paths will need Contents/Home for Mac maybe?

Reply:
I have my script include them in the JDK\_DIR being passed, which means this script stays generic, but the windows & ,mac scripts pass in the correct dir :)